arch/risc-v/include/common/irq.h

```diff
@@ -134,6 +140,14 @@
 #  error not supported !!!
```

Comment:
In 32 bit variant of include we had different vales for FPU registers.

Reply:
32-bit variant had
```
#if defined(CONFIG_ARCH_DPFPU)
#  define FPU_REG_SIZE      2
#elif defined(CONFIG_ARCH_QPFPU)
#  define FPU_REG_SIZE      4
#else
#  define FPU_REG_SIZE      1
#endif
```

Reply:
Yes, and CONFIG_ARCH_QPFPU had been removed since there are no actual support for qpfpu